paimon-core/src/main/java/org/apache/paimon/mergetree/compact/LookupChangelogMergeFunctionWrapper.java

```diff
@@ -85,6 +86,38 @@ public void add(KeyValue kv) {
     public ChangelogResult getResult() {
         reusedResult.reset();
 
+        if (mergeFunction.candidatesAllLevel0()) {
```

Comment:
Means there is no highlevel in mergeFunction?

It seems we should merge these two methods.